### PR TITLE
feat(map): quoting in multiple asset scales

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

- adds log to quote service
- fixes quote endpoint in docker compose
- allows MAP to quote in different asset scales, which is necessary to create quote with newly generated asset using admin api postman collection

## Context

- related to #516 

## Checklist

- [x] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [x] Make sure that all checks pass

### CODE REVIEW

File: packages/mock-account-provider/app/routes/quotes.ts [hunk 1]
```diff
@@ -35,31 +35,30 @@ export async function action({ request }: ActionArgs) {
   const feeStructure = CONFIG.seed.fees[0]
   const asset = CONFIG.seed.asset
   if (receivedQuote.paymentType == PaymentType.FixedDelivery) {
-    // TODO: handle quote fee calculation for different assets/scales
-    if (
-      receivedQuote.sendAmount.assetCode !== asset.code ||
-      receivedQuote.sendAmount.assetScale !== asset.scale
-    ) {
+    // TODO: handle quote fee calculation for different assets
+    if (receivedQuote.sendAmount.assetCode !== asset.code) {
```
Comment (mkurapov) [inline packages/mock-account-provider/app/routes/quotes.ts:39]: Should we check if `receivedQuote` properties exist at all?

**Review by mkurapov — APPROVED** [review]

File: infrastructure/local/seed.peer.yml [hunk 2]
```diff
@@ -22,5 +22,5 @@ accounts:
     path: accounts/planex
     postmanEnvVar: planexPaymentPointer
 fees:
-  - fixed: 100
+  - fixed: 1
```
[the remark this answers is not in the document]
Comment (sabineschaller) [reply infrastructure/local/seed.peer.yml:25]: It's not really a discount. It's just hidden 😉
```
feeStructure.fixed * Math.pow(10, receivedQuote.receiveAmount.assetScale))
```

File: packages/mock-account-provider/app/routes/quotes.ts (hunk 1 above)
Comment (sabineschaller) [reply packages/mock-account-provider/app/routes/quotes.ts:39]: It's the MAP. I don't think it needs these security checks. If the `receivedQuote` does not have its properties, something is broken in the backend and I sure hope we find out about that without the MAP.

Comment (wilsonianb) [reply packages/mock-account-provider/app/routes/quotes.ts:39]: >It's the MAP. 

:joy: 

- [ ] Tests added/updated (or it's just the MAP)

**Review by wilsonianb — APPROVED** [review]

File: packages/mock-account-provider/app/routes/quotes.ts (hunk 1 above)
Comment (sabineschaller) [reply packages/mock-account-provider/app/routes/quotes.ts:39]: The MAP doesn't even have tests...

**Pull request merged**